spring-cloud-skipper-client/src/main/java/org/springframework/cloud/skipper/client/DefaultSkipperClient.java

```diff
@@ -317,8 +320,8 @@ public Release rollback(String releaseName, int releaseVersion) {
 
 	@Override
 	public List<Release> list(String releaseNameLike) {
-		ParameterizedTypeReference<CollectionModel<Release>> typeReference = new ParameterizedTypeReference<CollectionModel<Release>>() {
-		};
+		ParameterizedTypeReference<HateoasResponseWrapper<ReleasesResponseWrapper>> typeReference =
```

Comment:
How is this different from using just the  `Release` domain object instead of explicit wrapper? I have the same question for other places where the ResponseWrappers are used.

Reply:
Server side is still sending hateoas json so wanted to completely remove hateoas classes from client. This is still needed for jackson.